src/main/java/org/gridsuite/study/server/service/VoltageInitResultConsumer.java

```diff
@@ -25,16 +30,36 @@ public class VoltageInitResultConsumer {
 
     private final ConsumerService consumerService;
 
-    public VoltageInitResultConsumer(StudyService studyService, ConsumerService consumerService) {
+    private final NotificationService notificationService;
+
+    public static final String HEADER_REACTIVE_SLACKS_OVER_THRESHOLD = "REACTIVE_SLACKS_OVER_THRESHOLD";
+    public static final String HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE = "reactiveSlacksThreshold";
+
+    public VoltageInitResultConsumer(StudyService studyService,
+                                     ConsumerService consumerService,
+                                     NotificationService notificationService) {
         this.studyService = studyService;
         this.consumerService = consumerService;
+        this.notificationService = notificationService;
+    }
+
+    private void checkReactiveSlacksOverThreshold(Message<String> msg, UUID studyUuid) {
+        consumerService.getNodeReceiver(msg).ifPresent(nodeReceiver -> {
+            Boolean alert = msg.getHeaders().get(HEADER_REACTIVE_SLACKS_OVER_THRESHOLD, Boolean.class);
+            if (Boolean.TRUE.equals(alert)) {
+                String userId = msg.getHeaders().get(HEADER_USER_ID, String.class);
+                Double alertThreshold = msg.getHeaders().get(HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE, Double.class);
+                notificationService.emitStudyAlert(studyUuid, nodeReceiver.getNodeUuid(), userId, new StudyAlert(AlertLevel.WARNING, "REACTIVE_SLACKS_OVER_THRESHOLD", Map.of("threshold", alertThreshold.toString())));
```

Comment:
use the static variable value instead of the string, sonar issue

Reply:
Done